native/game_backend/src/effect.rs

```diff
@@ -1,7 +1,7 @@
 use rustler::{NifMap, NifTaggedEnum};
 use serde::Deserialize;
 
-#[derive(Deserialize, NifMap, Clone)]
+#[derive(Deserialize, NifMap, Clone, Debug)]
```

Comment:
Do we want to keep all these Debug derives, or were them just to be able to print these struct during the debugging process?

Reply:
These Debug trait were added just for debugging while developing the list, i'll remove them since they're not necessary